bftengine/src/bftengine/ClientsManager.cpp

```diff
@@ -165,7 +165,7 @@ bool ClientsManager::isValidClient(NodeIdType clientId) const { return (clientId
 // * allocate new ClientReplyMsg
 // * calculate: num of pages, size of last page.
 // * save the reply to the reserved pages.
-std::unique_ptr<ClientReplyMsg> ClientsManager::allocateNewReplyMsgAndWriteToStorage(
```

Comment:
Update description of functions you changed.

Reply:
Fixed, thanks.